Task4/Task4_1/ExtensionMethods.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static Task4_1.Program;
+
+namespace Task4_1
+{
+    internal static class ExtensionMethods
```

Comment:
Better use generic Func<T, T, T> instead of SumDelegate

Reply:
Modified